Update README.md

ibenrodriguez opened the pull request:
typo in link to website - should be https://swarms.world not https://swarms.com

Although you could purchase https://swarms.com for $50k USD

ibenrodriguez committed:
Update README.md

typo in link to website - should be https://swarms.world not https://swarms.com

Although you could purchase https://swarms.com for $50k USD

evelynmitchell commented:
Thanks for the patch Iben. I notice there's a whitespace change on the logo line. Could you redo the patch to only include the URL change?

ibenrodriguez commented on README.md:
```diff
@@ -50,6 +50,6 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 
-![Swarms Logo](swarms_logo.png)
\ No newline at end of file
+![Swarms Logo](swarms_logo.png)
```
@evelynmitchell  - I've looked at line 55 different ways with different tools and don't see the whitespace - where is the whitespace change?  At the end of the line?

Can you let me know what tools you are using to see this?

evelynmitchell replied:
Its at the end of the line. 

I use vscode configured with markdownlint to spot whitespace changes.

This PR looks good to me.

ibenrodriguez reviewed (commented):
can't find the white space described by @evelynmitchell

ibenrodriguez committed:
added missing file for swarms_logo.png

added missing file for swarms_logo.png

found it here - https://github.com/The-Swarm-Corporation/Cookbook/blob/main/swarms_logo.png

evelynmitchell reviewed (commented):
Thanks for the PR!

Good to merge.

ibenrodriguez commented:
@kyegomez - can you merge this when you get a chance?
